samples/amber.cc

```diff
@@ -102,6 +105,33 @@ const char kUsage[] = R"(Usage: amber [options] SCRIPT [SCRIPTS...]
   -h                        -- This help text.
 )";
 
+// Parses a decimal integer from the given string, and writes it to |retval|.
+// Returns 1 if the parse was successful and consumed the whole string.
+static int ParseOneInt(const char* str, int* retval) {
+  char trailing = 0;
+  return
+#if defined(_MSC_VER)
+      sscanf_s(str, "%d%c", retval, &trailing, 1)
```

Comment:
I'd rather just see the `returns` put into the if's as it's less confusing to read.